ndk/src/looper.rs

```diff
@@ -269,7 +269,7 @@ impl ForeignLooper {
         &self,
         fd: RawFd,
         events: FdEvent,
-        callback: Box<F>,
+        callback: F,
```

Comment:
Doesn't make much difference I guess. If F is already a `Box<F': FnMut(RawFd) -> bool>` it'll be boxed twice. LGTM

Reply:
Hmm, I wasn't even aware that if `F: FnMut(RawFd) -> bool` then `Box<F>: FnMut(RawFd) -> bool` too. That makes this a non-breaking change, while at the same time not requiring users to manually wrap the thing in `Box::new`. It's anyway curious that we didn't do this before, despite the requirement to `Box` the `dyn` twice (receiving a `Box::into_raw` pointer on a `dyn` object obviously returns a fat-pointer, which does not fit in a `*mut c_void`). We could have just done `Box::new(Box::new())` before... Oh well.